### PR TITLE
Added tooltip when using organisation filter for the first time

### PR DESCRIPTION
**Pull request by AlexGilleran**
### What this PR does
All the things in #1447. Principally:
- Removes search filters from mobile
- Makes it so that if you search for something on mobile and you have a search filter active (from following a URL), it strips all the filters
- Makes it so that if you follow a link to datasets by an organisation on desktop, it shows you a tooltip explaining that a filter is active. Once you click away it records that you've seen it in localstorage so that it doesn't show again on that computer
- To make this work, there's a new tooltip component that is used by both the new tooltip and the old data quality one. This works way more dynamically, using javascript to centre itself. I also changed the colour of both tooltips to purple, which was @philipgrimmett approved ™️ .

### Checklist

-   [x] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column
-   [x] There are sufficient comments for my code to be understandable - and I realise reviewers will pull me up on it if not!

### CODE REVIEW

File: magda-web-client/src/UI/Tooltip.js [hunk 1]
```diff
@@ -1,24 +1,110 @@
...
+
+    componentDidMount() {
+        if (this.props.requireClickToDismiss) {
+            document.addEventListener("mousedown", this.handleClickOutside);
+        }
+        this.adjustOffset();
+    }
+
+    componentDidUpdate() {
+        this.adjustOffset();
```
Comment (chloeleichen) [inline magda-web-client/src/UI/Tooltip.js]: change state here cause an infinite loop, eg, if you go to http://localhost:6108/dataset/ds-dap-csiro%3A17050/details?q=water and wait for a few seconds

**Comment by chloeleichen** [comment]
1. new tooltip component looks good & works well
2. >  if you follow a link to datasets by an organisation on desktop, it shows you a tooltip explaining that a filter is active
I can't actually find it, can you upload a screenshot?

**Comment by AlexGilleran** [comment]
Yeah it's not working anymore :'(

**Comment by AlexGilleran** [comment]
Better again now - want to give it one last look @chloeleichen ?

**Comment by chloeleichen** [comment]
Eh I must be looking at the wrong thing :(. I still don't see the tooltip. Maybe you can show me when I'm in tomorrow

**Pull request merged**